system-parachains/asset-hubs/asset-hub-polkadot/src/ah_migration/mod.rs

```diff
@@ -408,9 +411,49 @@ impl RcToAhCall {
 					);
 				})?;
 
+				// Convert the versioned XCM to latest version for DescendOrigin conversion
+				let xcm: xcm::latest::Xcm<()> = (*message).try_into().map_err(|err| {
```

Comment:
Why did we not do this for Kusama? Was it missing?

Reply:
However, I decided it was too complicated to map this pattern. I'll advise any new proposal to use `AliasOrigin` directly instead. Since it would end up being mapped to AH anyway.